### ISSUE TITLE
[TextFields] Hidden labels (clear text) may still be accessible to VoiceOver

### ISSUE THREAD

**Issue by romoore** [material-components/material-components-ios#1848]
### Overview
We need to quickly audit/review text labels to ensure that various hidden labels are not being exposed to accessibility users if they are not visible.

### Reproduction steps
1. Launch the MDC Catalog and turn on VoiceOver
1. Make sure the only accessibility elements available to VoiceOver users are the ones visible on-screen for text fields (and full-width text fields).

### Expected behavior

Only visible labels are read by VoiceOver and only once.

### Actual behavior

The placeholder is read by VoiceOver twice.

### Version number
31.0.2

### Operating system and device
Any/all

---
#### Internal data
- Associated internal bug: [b/117179427](http://b/117179427)

**Comment by romoore** [comment]
Looks like MultiLine text fields might have a bug around the placeholder label.

**Comment by material-mirror** [comment]
The internal issue [b/117179427](http://b/117179427) is now closed. This issue is being closed as a result.
